virtcontainers/qemu_arch_base.go

```diff
@@ -432,15 +432,8 @@ func (q *qemuArchBase) appendVSockPCI(devices []govmmQemu.Device, vsock kataVSOC
 
 func networkModelToQemuType(model NetInterworkingModel) govmmQemu.NetDeviceType {
 	switch model {
-	case NetXConnectBridgedModel:
-		return govmmQemu.MACVTAP //TODO: We should rename MACVTAP to .NET_FD
 	case NetXConnectMacVtapModel:
 		return govmmQemu.MACVTAP
-	//case ModelEnlightened:
```

Comment:
Note: This is already implemented implicitly today. The interconnect model is only used today when the interface cannot be directly connected to the VM. The vhost-user interface is also handed using a special case and works with only a certain class of vhost-user plugins.

So our default model is actually ModelEnlightened when running in tcFilter or macvtap mode. The none mode is the only one that does not behave this way.